<!-- Launches: path_planner_in_occupancy_grid -->
<launch>
	
	<arg name="drone_id_namespace"    default="drone11" />
	<arg name="move_base_goal_topic"  default="move_base_simple/goal" />
	<arg name="move_base_path_topic"  default="move_base/NavfnROS/plan" />
	<arg name="output_path_topic" 	  default="path_with_id" />
	<arg name="generate_path_service" default="generate_path" />
	<arg name="debug"		  default="false" />
	<arg name="timeout"		  default="10" />
	<!-- timeout in seconds -->

	<group ns="$(arg drone_id_namespace)">
		<node name="path_planner_in_occupancy_grid"
					pkg="path_planner_in_occupancy_grid"
					type="path_planner_in_occupancy_grid"
					output="screen">
	    	<param name="move_base_goal_topic" 	value="$(arg move_base_goal_topic)"	 type="str"   />
			<param name="move_base_path_topic" 	value="$(arg move_base_path_topic)"	 type="str"   />
			<param name="output_path_topic" 	value="$(arg output_path_topic)"         type="str"   />
			<param name="generate_path_service"     value="$(arg generate_path_service)"     type="str"   />
			<param name="debug"			value="$(arg debug)" 			 type="bool"  />
			<param name="timeout"			value="$(arg timeout)"			 type="double"/>
		</node>

	</group>
</launch>
